### DISCUSSION TITLE
Is this font abandoned?

### DISCUSSION THREAD

**Discussion by victorbombella** [tonsky/FiraCode#1282] [General]
Hi all!

No offense, just asking. Is this font abandoned? My doubt arise since the last release (5.2) was over one year ago. Even, there are, at least, three unmerged pull requests from 2020...

So, the question is... is this font alive?

Regards!

**Comment by tonsky** [comment]
Hi, it’s not abandoned. I’ve been planning v6 for a few months, many of the commits are in the master already. But it’s slow and I’ve been distracted. Soon :)
